server/test/Epics.test.ts

```diff
@@ -82,27 +82,52 @@ beforeAll(async () => {
 
 afterAll(async () => {
   const connection = await getConnection();
-  await connection.query(`DELETE FROM ORGANIZATIONS;`);
-  await connection.query(`DELETE FROM PROJECTS;`);
   await connection.query(`DELETE FROM USERS;`);
   await connection.query(`DELETE FROM EPICS;`);
+  await connection.query(`DELETE FROM PROJECTS;`);
+  await connection.query(`DELETE FROM ORGANIZATIONS;`);
+  await connection.query('ALTER TABLE ORGANIZATIONS AUTO_INCREMENT = 1');
+  await connection.query('ALTER TABLE USERS AUTO_INCREMENT = 1');
+  await connection.query('ALTER TABLE PROJECTS AUTO_INCREMENT = 1');
+  await connection.query('ALTER TABLE EPICS AUTO_INCREMENT = 1');
   await connection.close();
 });
 
+let Cookies = '';
+
+beforeEach((done) => {
+  request(app)
+    .post('/api/users/login')
+    .send({
+      email: 'test@test.com',
+      password: 'token',
+    })
+    .end(function (err, res) {
+      Cookies = res.headers['set-cookie'].pop().split(';')[0];
+      done();
+    });
+});
+
 // FIXME: 세션에 사용자 mock
 test('전체 에픽 목록을 조회한다.', async () => {
-  const res = await request(app).get('/api/epics?projectId=1');
+  const req = request(app).get('/api/epics?projectId=1');
+  req.set('Cookie', Cookies);
+  const res = await req;
```

Comment:
이러면 그냥 이렇게 써도 되지 않나요??

```suggestion
  const res = await request(app)
    .set('Cookie', cookies)
    .get('/api/epics?projectId=1');
```